components/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/token/AbstractJWTGenerator.java

```diff
@@ -0,0 +1,633 @@
+/*
```

Comment:
1. We need to re-write the this class with only JWT stuff.
2. Need to conform with the JWT spec (e.g. alg, sub) 
3. Multiple algoritm support